services/db_shell/fixtures/example.cypher

```diff
@@ -1,27 +1,28 @@
-// Set up basic client references
+--// Set up basic client references
 CREATE (:User{uid: 1});
 CREATE (:User{uid: 2});
 
 // Syntax to log a user measurement at a certain time
 MATCH (u:User{uid:1})
-    MERGE (m:Measurement{type: "sentiment", value: 0.5}) 
-            -[:RecordedAt{time: 1000}]-> (d:Date{day:1, month:1, year:2021})
-    MERGE (m) -[:RecordedBy]-> (u)
-    MERGE (d) -[:Includes]-> (m)
-    MERGE (u) -[:Recorded]-> (m);
+    MERGE (m:Measurement{type: "sentiment"}) -[:Includes]-> (v:MeasurementValue{value: 0.5})
+    MERGE (s:Source{type: "app"}) -[:Includes]-> (m)
+    MERGE (u) -[:Recorded]-> (m)
+    MERGE (v) -[:RecordedOn]-> (d:Date{day:1, month:1, year:2021})
+    MERGE (v) -[:RecordedAt]-> (h:Hour{time: 6175});
+
 
 // An additional record for the same user on a different day
 MATCH (u:User{uid:1})
-    MERGE (m:Measurement{type: "sentiment", value: 0.7}) 
-            -[:RecordedAt{time: 2100}]-> (d:Date{day:2, month:1, year:2021})
-    MERGE (m) -[:RecordedBy]-> (u)
-    MERGE (d) -[:Includes]-> (m)
-    MERGE (u) -[:Recorded]-> (m);
+    MERGE (m:Measurement{type: "sentiment"}) -[:Includes]-> (v:MeasurementValue{value: 0.7})
+    MERGE (s:Source{type: "app"}) -[:Includes]-> (m)
+    MERGE (u) -[:Recorded]-> (m)
+    MERGE (v) -[:RecordedOn]-> (d:Date{day:2, month:1, year:2021})
+    MERGE (v) -[:RecordedAt]-> (h:Hour{time: 6175});
 
 // Different user, different type of measurement
 MATCH (u:User{uid:2})
-    MERGE (m:Measurement{type: "heart-rate", value: 70}) 
-            -[:RecordedAt{time: 0}]-> (d:Date{day:1, month:1, year:2021})
-    MERGE (m) -[:RecordedBy]-> (u)
-    MERGE (d) -[:Includes]-> (m)
-    MERGE (u) -[:Recorded]-> (m);
\ No newline at end of file
+    MERGE (m:Measurement{type: "heart-rate"}) -[:Includes]-> (v:MeasurementValue{value: 70})
+    MERGE (s:Source{type: "app"}) -[:Includes]-> (m)
+    MERGE (u) -[:Recorded]-> (m)
+    MERGE (v) -[:RecordedOn]-> (d:Date{day:1, month:1, year:2021})
+    MERGE (v) -[:RecordedAt]-> (h:Hour{time: 6175});
```

Comment:
thanks for keeping this updated. Shouldn't we update the test cypher as well? Also, see my comments in the OGM PR related to fixtures - we may want to stop maintaining the cypher files.

Reply:
That makes sense. Could we make a new PR dedicated to having a file that loads fixtures by running Neogma?

Reply:
Works for me